app/controllers/teachers/reviews_controller.rb

```diff
@@ -29,8 +29,9 @@ def update
     end
 
     def destroy
+      student_id = @review[:student_id]
```

Comment:
Here as well. Why do you treat `@review` as a hash?

Reply:
Same here, this line is not required